### ISSUE TITLE
[CommandBar] buttons are mis-aligned

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#4906]
It looks like the command bar buttons recently shifted a bit up and are no longer aligned in the center with the toggle button.

<img src='https://shipusercontent.com/d6a26e69bbbab527b023935a36814481/Screen%20Shot%202017-12-13%20at%2011.07.18%20AM.png' title='Screen Shot 2017-12-13 at 11.07.18 AM.png' width=376>

**Comment by ryanjduffy** [comment]
Looks like the toggle button is (loosely?) centered with the tabs whereas the command bar buttons are (loosely?) centered in the bar. To which should they align?

I'd like to do a bit of refactoring to standardize these components to ensure some consistency across their usage.

**Comment by lukaszsobek** [comment]
@ryanjduffy if you remove this class
`.command-bar > button {
   padding: 6px 5px;
}`
they will be perfectly aligned again. Alternatively you can also add it to:
`.toggle-button-start.collapsed, .toggle-button-end.collapsed`
which will have a similar effect

**Comment by ryanjduffy** [comment]
@lukaszsobek Thanks! I saw the same. While that does produce the right result, I think we can improve the layout. After removing that rule, the button has an 8px top/bottom margin leaving 13px for height while the contained image is actually 16px. So we're overflowing the button's content box and effectively treating it like it has a top padding of 8px and bottom padding of 5px.

However, none of the padding gymnastics is really even necessary because both the editor pane and the command bar is using flex layout. I refactored the shared button component out and reworked the CSS to simplify it and rely on the flex layout. I also adjusted the resume SVG which was offset to the right of the viewbox. Here's gif of my change compared to just removing the padding rule.

![4809-diff](https://user-images.githubusercontent.com/788456/34007599-341743f4-e0b7-11e7-8b58-fe036714a324.gif)
